site/src/atomics/shadow.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Shadow Atomics
+description: Shadow related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Shadow Atomics
+
+Shadow atomics are used to add a shadow effect around the frame of an element.
+
+## Usage
+
+Here are examples of shadow atomics, ranging from `none` for no shadow to `extra-heavy` shadow.
+
+```html
+<div class="display-block border padding-block-sm">
+	<div class="box-shadow-none margin-block-xs">box-shadow-none</div>
```

Comment:
Can we jujsh these examples up just a tiny bit with some padding and larger text?

Reply:
Like this? 

![image](https://user-images.githubusercontent.com/94572161/147994845-4fe69e64-dd89-4373-b727-c002951a0047.png)

Reply:
Updated, need to keep some margin to make sure the shadows don't collide with the next example.

![image](https://user-images.githubusercontent.com/94572161/148112005-6e473e43-e4a7-46a6-aa30-07a74bf59b93.png)